bbc/simorgh
# Fix brand links for services with variants

Overall changes
======
- Since removing the redirect to preferred variant logic, the brand logo home link no longer works as expected if the user is on a service variant that isn't the 'main' variant
- Ensures `uzbek` returns `/uzbek` in all cases, as they do not yet publish variant homepages

Testing
======
1. Visit http://localhost:7080/serbian/cyr/srbija-68707945?renderer_env=live
2. Hover over the brand logo and confirm the URL is `/serbian/cyr`
3. Click the link and confirm it takes you to `/serbian/cyr`
4. Visit http://localhost:7080/serbian/lat/srbija-68707945?renderer_env=live
5. Hover over the brand logo and confirm the URL is `/serbian/lat` 
6. Click the link and confirm it takes you to `/serbian/lat`

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
